src/common/components/clusterWizard/ClusterWizardStep.tsx

```diff
@@ -15,6 +15,10 @@ const ClusterWizardStep: React.FC<ClusterWizardStepProps> = ({ navigation, foote
     <div className={css(styles.wizardOuterWrap, 'cluster-wizard-step')}>
       <div className={css(styles.wizardInnerWrap)}>
         {navigation}
+        {
+          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+          /* @ts-ignore: the prop should be optional depending on the WizardBody usage */
+        }
```

Comment:
why is this needed now ? 
We should disable the ts check as it can result in runtime errors

Reply:
This rule is added by Jirka in this PR :https://github.com/openshift-assisted/assisted-ui-lib/pull/1472/files#diff-551d01961ef7ef4a0587a71ba46ce7286af5b39a0ec189e7b0dcfdfb7ceb6224 because if not the test failed